src/types/ExternalImage.ts

```diff
@@ -0,0 +1,9 @@
+type ExternalImage = {
+    "albumId": number,
+    "id": number,
+    "title": string,
+    "url": string,
+    "thumbnailUrl": string,
+};
+
+export default ExternalImage;
```

Comment:
These don't need default exports. You can just `export type ExternalImage` if you want to export it.